{"links": {"self": {"href": "data/repositories/osrf/drcsim/issues/214/comments/3843467.json"}, "html": {"href": "#!/osrf/drcsim/issues/214#comment-3843467"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/drcsim/issues/214.json"}}, "type": "issue", "id": 214, "repository": {"links": {"self": {"href": "data/repositories/osrf/drcsim.json"}, "html": {"href": "#!/osrf/drcsim"}, "avatar": {"href": "data/bytebucket.org/ravatar/{247b5408-ae91-4e3c-b2f9-a21994658d11}ts=c_plus_plus"}}, "type": "repository", "name": "drcsim", "full_name": "osrf/drcsim", "uuid": "{247b5408-ae91-4e3c-b2f9-a21994658d11}"}, "title": "atlas does not walk with 5steps.py"}, "content": {"raw": null, "markup": "markdown", "html": "", "type": "rendered"}, "created_on": "2013-04-23T11:31:51.617832+00:00", "user": {"display_name": "Masaki Murooka", "uuid": "{1f296b4c-43e3-4458-ae6e-5e47274682c1}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B1f296b4c-43e3-4458-ae6e-5e47274682c1%7D"}, "html": {"href": "https://bitbucket.org/%7B1f296b4c-43e3-4458-ae6e-5e47274682c1%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/a27173c4d7224b095d320a3d3c4b8e95d=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsMM-6.png"}}, "nickname": "mmurooka", "type": "user", "account_id": "5ac5e2d16f48f84dd9998e8b"}, "updated_on": null, "type": "issue_comment", "id": 3843467}